scripts/update-pkgs-2021.12.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$EPREFIX" ]; then
+    # this assumes we're running in a Gentoo Prefix environment
+    EPREFIX=$(dirname $(dirname $SHELL))
+fi
+
+# first update checkout of gentoo repository to sufficiently recent commit
+# this is required because we pin to a specific commit when bootstrapping the compat layer
+# see gentoo_git_commit in ansible/playbooks/roles/compatibility_layer/defaults/main.yml
+cd $EPREFIX/var/db/repos/gentoo
+git fetch origin
+git checkout d7c647404a632309810851c52c1d350cafc26949  # 2022-08-25
+cd -
+
+# unmask GCC 9.x, since we're using that as system compiler in EESSI pilot 2021.12
+sed -i 's@<sys-devel/gcc-[0-9]*@<sys-devel/gcc-9@g' $EPREFIX/var/db/repos/gentoo/profiles/package.mask
```

Comment:
This is not the place to do this. You can just add `sys-devel/gcc:9` to `$EPREFIX/etc/portage/package.unmask`.

Reply:
fixed in 0c8fcc4